frontend/src/components/Comments/CommentForm.vue

```diff
@@ -1,41 +1,68 @@
 <template>
-    <CardForm :title="`Submit ${type}`">
-      <textarea placeholder="Your input here.." v-model="body"></textarea><br>
+    <Card>
+      <div class="title-container">
+        <h4>{{ title }}</h4>
+        <aside v-if="closeCallback" v-on:click="closeCallback"><i class="material-icons">close_circle</i></aside>
+      </div>
+      <textarea placeholder="" v-model="body"></textarea><br>
       <!-- should probs be a separate component -->
       <AppButtonToolTip
         @click.native="callback({body}); body='';"
         :disabled="!authenticated"
         :disabledMessage="disabledMessage">
-        {{ type }}
+        Submit
       </AppButtonToolTip>
       <!-- errors will be injected here -->
       <slot></slot>
-  </CardForm>
+  </Card>
 </template>
 
 <script>
-import CardForm from '@/components/Card/Form'
+import Card from '@/components/Card'
 import AppButtonToolTip from '@/components/AppButton/WithToolTip'
 
 export default {
   name: 'CommentForm',
   components: {
-    CardForm,
+    Card,
     AppButtonToolTip
   },
   props: {
     type: String,
+    title: String,
     callback: Function,
-    authenticated: Boolean
+    authenticated: Boolean,
+    closeCallback: Function
   },
   data () {
     return {
       body: '',
       disabledMessage: {
-        content: 'You must be to logged in for this functionality.',
+        content: 'You must be logged in for this functionality.',
         placement: 'right'
       }
     }
   }
 }
 </script>
+
+<style scoped>
+h4 {
+  font-weight: normal;
```

Comment:
we should amend the font-matrix if there's nothing that matches desired size.

Reply:
body-copy-1 works